### ISSUE TITLE
Consider removing Oracle JDK as of JDK 11?

### ISSUE THREAD

**Issue by whaley** [skotchpine/asdf-java#33]
Oracle's JDK has new license terms that effectively prevent it from being used for anything other than development ([here is a pretty decent summary from the author the joda-* libraries and the java.time API](https://blog.joda.org/2018/09/do-not-fall-into-oracles-java-11-trap.html)).

I doubt this would be an actual concern since asdf seems to be primarily used for development environments, at least for Java tooling.  However, since the general advice is to *not* use Oracle's JDK unless you pay them for commercial support, we should consider removing the Oracle distribution as an option.

**Comment by x80486** [comment]
Not quite. There are many companies that are willing to pay for that. This is just a tool to install, extremely easy, the Java JDKs.

**Comment by skotchpine** [comment]
Hey, @whaley, thanks for bringing this up. I've been wondering what we should do.

What do people think about having a very clear opt-in action?
i.e. "if you set `$ORACLE_ISNT_THE_DEVIL_TO_ME`, then `oracle-11.0.1` is available to you"

**Comment by skotchpine** [comment]
Wanna rename this to something more like "What to do about Oracle's license"

**Comment by mig4** [comment]
Well, as @whaley mentioned it is still valid to use Oracle's distribution for testing and development (point B in supplemental terms) and even in production if you have a license from Oracle. We'll still probably see it used in CI systems as one of the JDKs to test against, etc.

So I'd personally lean towards displaying it as an option as usual, and just adding a post-install message saying if you plan to use this installation for anything other than testing or development you need to obtain license from Oracle, and link to https://www.oracle.com/technetwork/java/javase/terms/license/index.html
